src/main/java/com/microsoft/aad/msal4j/JsonHelper.java

```diff
@@ -29,4 +31,73 @@
             throw new MsalClientException(e);
         }
     }
+
+    /**
+     * Merges given JSON strings into one Jackson JSONNode object, which is returned as a String
+     */
+    static String mergeJSONString(String mainJsonString, String addJsonString) {
+        JsonNode mainJson, addJson;
```

Comment:
nit: split these into two lines. 

For the time being, I've been using the [Google Java style](https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line) for guidance. We should all agree on guidelines that we agree on and ideally add a linter to the build.